LFS-976: Wrong date format shown on the patient charts

sdumitriu opened the pull request.

sdumitriu committed:
LFS-976: Wrong date format shown on the patient charts

marta- commented:
Did you update the spreadsheet and run the import script?

sdumitriu commented:
No.

sashaandjic commented:
tested the code change

acrowthe reviewed (changes requested):
Missing `DD/MM/yyyy` in Graded Walk Test.xml.

Otherwise, everything looks good

sdumitriu commented:
> Missing `DD/MM/yyyy` in Graded Walk Test.xml.

Shouldn't it be `MM/dd/yyyy`? It's the only one with this order.

sdumitriu committed:
LFS-976: Wrong date format shown on the patient charts
Missed one date due to wrong order

acrowthe committed:
LFS-979: Cardiac rehab importer should fix incorrect date format capitalization
Automatically change the following characters in dateFormats:
- D to d (as users are unlikely to want day of year)
- Y to y (as week year is not likely desired and it is easy to miss accidental uses)
- `-` to `/` (for consistancy between specified dates)

sashaandjic commented:
retested after the merge
